ui/src/views/mint/github-step/steps/github-repo-configuration/repo-branch-commit-fields.tsx

```diff
@@ -54,7 +67,8 @@ export const RepoBranchCommitFields = () => {
     <>
       <Form.Field>
         <Form.Label>Git Branch</Form.Label>
-        <Dropdown
+        <Combobox
+          // TODO replace left icon with branch icon

```

Comment:
I was thinking to have this
<img width="480" alt="image" src="https://user-images.githubusercontent.com/52458215/224811879-84fec1b2-54e2-46c4-a38d-cf73922d6d8d.png">


Instead of this 
<img width="478" alt="image" src="https://user-images.githubusercontent.com/52458215/224808127-4f5da0c0-0e7f-405c-914c-895994d9f898.png">